internal/pkg/workspace/workspace.go

```diff
@@ -151,6 +153,52 @@ func (ws *Workspace) ListWorkloads() ([]string, error) {
 	})
 }
 
+// ListPipelines returns the names of all pipelines in the workspace.
+func (ws *Workspace) ListPipelines() (map[string]string, error) {
```

Comment:
Maybe this could return to an array of object as opposed to a map? Because map is not helpful as an API output for the users to find out what the key-value means.